Carbon.Sample.API/Properties/launchSettings.json

```diff
@@ -1,34 +1,17 @@
 {
-  "iisSettings": {
-    "windowsAuthentication": false,
-    "anonymousAuthentication": true,
-    "iisExpress": {
-      "applicationUrl": "http://localhost:35418",
-      "sslPort": 44352
-    }
-  },
   "$schema": "http://json.schemastore.org/launchsettings.json",
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
-      "launchBrowser": true,
-      "launchUrl": "swagger",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
-    },
     "Carbon.Sample.API": {
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
       "commandName": "Project",
-      "launchBrowser": true,
-      "launchUrl": "swagger",
       "environmentVariables": {
         "CONFIGURATION_TYPE": "FILE",
-        "FILE_CONFIG_PATHS": "appsettings.Development.json",
+        "FILE_CONFIG_PATHS": "appsettings.kube_test.json",
```

Comment:
Proje içerisinde bir app setting bırakalım; bu appsettingsde şifre, url vb olmamalı. Ortamlar, uygulama nasıl debug edilir neler gereklidir bunun için ayrı bir eğitim içeriği olmalı.